pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -64,6 +66,14 @@ def on_trainer_init(
         self.trainer.flush_logs_every_n_steps = flush_logs_every_n_steps
         self.trainer.log_every_n_steps = log_every_n_steps
         self.trainer.move_metrics_to_cpu = move_metrics_to_cpu
+        if not isinstance(self.trainer.logger, LoggerCollection) and is_overridden(
+            "agg_and_log_metrics", self.trainer.logger, LightningLoggerBase
+        ):
+            self._override_agg_and_log_metrics = True
+            rank_zero_deprecation(
+                "`LightningLoggerBase.agg_and_log_metrics` is deprecated in v1.6 and will be"
+                " removed in v1.8. Please use `LightningLoggerBase.log_metrics` instead."
```

Comment:
This should mention that the Trainer will directly call `LightningLoggerBase.log_metrics`

The "user" using the API is the Trainer. The developer may have implemented a logger which implements both log_Metrics & agg_and_log_metrics and would be confused by this

Reply:
Good point, I'll clarify the error message